helm/dagster/schema/schema_tests/test_scheduler.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+import yaml
+from kubernetes.client import models
+from schema.charts.dagster.subschema.scheduler import (
+    DaemonSchedulerConfig,
+    Scheduler,
+    SchedulerConfig,
+    SchedulerType,
+)
+from schema.charts.dagster.values import DagsterHelmValues
+from schema.utils.helm_template import HelmTemplate
+
+
+@pytest.fixture(name="template")
+def helm_template() -> HelmTemplate:
+    return HelmTemplate(
+        helm_dir_path="helm/dagster",
+        subchart_paths=["charts/dagster-user-deployments"],
+        output="templates/configmap-instance.yaml",
+        model=models.V1ConfigMap,
+        namespace="test-namespace",
+    )
+
+
+def test_scheduler_name(template: HelmTemplate):
```

Comment:
+ delete commented out lines?
```suggestion
def test_scheduler_config(template: HelmTemplate):
```